mdns/dllmain.c

```diff
@@ -34,80 +34,3 @@ BOOL APIENTRY	DllMain( HANDLE inModule, DWORD inReason, LPVOID inReserved )
 	}
     return( TRUE );
 }
-
-
-BOOL
-IsSystemServiceDisabled()
```

Comment:
Hi Marc, from my point of view, it is simply wrong to have any other code in dllmain.c. Also the function "IsSystemServiceDisabled" is only needed as a helper function in "mdns/dnssd_clientstub.c"; therefor I also made it static.
Of course from the point of comparability with older version, it would be better to keep the function in the same place...
I did not change anything in the function (except the "static" keyword"). I would prefer to move the function to the correct place.